### ISSUE TITLE
Audio capture doesn't work with non-default sink

### ISSUE THREAD

**Issue by Slater91** [dimtpap/obs-pipewire-audio-capture#40]
Hello! I have noticed that when a non-default sink is selected, or if the sink is changed after OBS is opened, audio isn't recorded at all (and I noticed the hard way: I recorded a whole interview with no audio...). This is reproducible 100% of the time and with any sink. Sometimes the issue is seen when changing the sink after OBS is started, sometimes even if it is changed _before_ OBS is started, which makes understanding what's going on really difficult.

I am using PipeWire 1.0 with WirePlumber (no difference is seen when using pipewire-media-session) from the PipeWire PPA related to [this project](https://github.com/pipewire-debian/pipewire-debian/) on KDE Neon (based on Ubuntu 22.04).

Here are the OBS logs:
[2023-12-05 11-29-09.txt](https://github.com/dimtpap/obs-pipewire-audio-capture/files/13559497/2023-12-05.11-29-09.txt)

Here is the output of pw-dump:
[pw-dump.json](https://github.com/dimtpap/obs-pipewire-audio-capture/files/13559502/pw-dump.json)

Here is a screen recording that showcases the issue:

https://github.com/dimtpap/obs-pipewire-audio-capture/assets/11941225/6c56d3e2-0c5d-4e45-a1d6-b4b13e40d76a

Any idea what might be going on?

**Comment by dimtpap** [comment]
I can't reproduce this. Were the sources in the video set to "Default"?

**Comment by Slater91** [comment]
Yes, they were set to "default". Nothing changes if I manually select the source.

**Comment by dimtpap** [comment]
Are you on the latest version? (There's no way to check, sorry, if you don't know just install the latest one)

**Comment by Slater91** [comment]
Sorry for my late reply. I have just updated the plugin and on the latest version it still displays the same behaviour.
